app/models/booking.rb

```diff
@@ -0,0 +1,5 @@
+class Booking < ApplicationRecord
+  belongs_to :animal
+  belongs_to :user
+  validates :date, :start_time, :duration, :total_fee, :animal_id, :user_id, presence: true
```

Comment:
would validate :user instead of :user_id, saves some code and might make it look cleaner :)

same for animal

Reply:
Removed those to make them more syntactically clean